bin/reth/src/cli/mod.rs

```diff
@@ -271,6 +271,9 @@ mod tests {
     /// Tests that the log directory is parsed correctly. It's always tied to the specific chain's
     /// name
     #[test]
+    #[cfg(not(feature = "telos"))]
```

Comment:
I don't think we need to disable the upstream tests individually like this, there will be a lot of tests that will not work with our changes and we want to make as few changes as possible (to reduce merge conflicts).  I have removed the .github directory and added our own CI workflow which will just run our E2E test.